worldengine/draw.py

```diff
@@ -693,7 +696,7 @@ def draw_scatter_plot(world, size, target):
         for x in range(world.width):
             if world.is_land((x, y)):
                 t = world.temperature_at((x, y))
-                p = world.humidity['data'][y, x]
+                p = world.humidity_at((x, y))
```

Comment:
But is there any chance at all that `humidity_at((x, y))` will ever not return the same value as `humidity['data'][y, x]`? For something like `is_land()` that might be the case, since the decision could eventually have some layers - but for most variables I don't see that being the case.

Calls to numpy-functions usually undermine the concept of maintainability anyway, the arrays are directly used instead of via a call to a centralized function. So maintainability in the face of renaming variables or restructuring the code isn't given either - with none of the notations.

In short: I understand the reasoning, but I don't see a realistic case where the methodology would ever pay off. And until then it is going to eat performance.